src/main/java/io/zold/api/Wallet.java

```diff
@@ -84,44 +79,31 @@
         /**
          * Path of this wallet.
          */
-        private final IoCheckedScalar<Path> path;
+        private final Path path;
 
         /**
          * Ctor.
          * @param path Path of wallet
          */
         File(final Path path) {
-            this(() -> path);
-        }
-
-        /**
-         * Ctor.
-         * @param pth Path of wallet
-         */
-        File(final Scalar<Path> pth) {
-            this.path = new IoCheckedScalar<>(
-                new SyncScalar<>(
```

Comment:
@llorllale We wrapped it to the `SyncScalar` to avoid the issues of simultaneous access to the same file from different threads. If we don't wrap it we are at risk of getting errors due to the fact, that OS won't allow us to access files, that are being held.

Reply:
@t-izbassar I believe the `Wallet` itself should be synced/stickied. Don't you think?

Reply:
@llorllale yes, that might be an option as well.